net/znc/Makefile

```diff
@@ -31,6 +31,7 @@ define Package/znc/default
   CATEGORY:=Network
   TITLE:=ZNC
   URL:=http://en.znc.in/
+  USERID:=znc=65532:znc=65532
```

Comment:
Personally I'd prefer to keep unprivileged 'system/daemon' users out of the way and close to other similar user accounts...e.g. nobody.  Whilst I can see your logic in '6667', I don't like it 'cos it's not low enough to scream 'system ID' at me and not high enough to be 'out of the way'  - but if that's the difference between accept/reject I really don't care.